converse_quiz.js

```diff
@@ -0,0 +1,91 @@
+/*
+The purpose of this challenge is to create a quiz and test the users knowledge.
+1. Must ask 5 questions
+2. keep track of correct questions
+3. Let user know # of question they answered correctly.
+4. Rank user depending on how many questions they answered correctly.
+
+
+I chose to choose questions based on Converse history.
```

Comment:
Grammatical error here. Maybe better "These are a few basic questions over the history of Converse." Little less wordy.